pkg/container/container.go

```diff
@@ -298,9 +301,5 @@ func (c Container) VerifyConfiguration() error {
 		return errorInvalidConfig
 	}
 
-	if len(hostConfig.PortBindings) > 0 && containerConfig.ExposedPorts == nil {
```

Comment:
Containers get recreated with the correct ports for me, so I'm assuming the Docker client uses the ones it finds in `hostConfig.PortBindings` and doesn't necessarily need those from `*container.Config.ExposedPorts`

Reply:
If we did this instead:
```go
if len(hostConfig.PortBindings) > 0 && containerConfig.ExposedPorts == nil {
	containerConfig.ExposedPorts = make(map[Port]struct{})
}
```
it would just align the docker responses for synology and non-synology docker daemons.